tests/src/main/java/testcases/KT007_scope_factory_dependencies/Scope.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package testcases.KT007_scope_factory_dependencies
+
+import motif.Creatable
+
+@motif.Scope
+interface Scope : Creatable<Scope.Dependencies> {
+    // note the absence of string() accessor (e.g. sink) in this scope
```

Comment:
Not having an accessor for `string()` here is important otherwise the value of `getDependencyMethodData(scope).isNotEmpty()` in `ScopeImplFactory` will evaluate to true (which is why the test passed in the original pull request)